accounts/abi/bind/base.go

```diff
@@ -406,6 +406,10 @@ func (c *BoundContract) WatchLogs(opts *WatchOpts, name string, query ...[]inter
 
 // UnpackLog unpacks a retrieved log into the provided output structure.
 func (c *BoundContract) UnpackLog(out interface{}, event string, log types.Log) error {
+	if log.Topics[0] != c.abi.Events[event].ID {
+		return fmt.Errorf("event signature mismatch")
+	}
+
```

Comment:
Can you please remove the blank line?

Reply:
sure. [a6852d4](https://github.com/ethereum/go-ethereum/pull/23230/commits/a6852d4e16e3512c578697a38cc9373531e6c4fb)